build/python.props

```diff
@@ -1,5 +1,5 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="3.0.14492" />
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="3.1.1.7" />
```

Comment:
is this the right  version? release notes refer to 1.1.7 : https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.7 ?

Reply:
Yes, because we use different packages on v2 / v3, and recently we change our version scheme to reflect which Python worker version we use. This is the correct version.

Reply:
I will go ahead an approve the PR. But please consider generating  release tags that match the nuget package version here.